### PR TITLE
Added android channel id to notification struct

### PR DESCRIPTION
**Pull request by hugoghx**

### CODE REVIEW

File: gcm.go [hunk 1]
```diff
@@ -116,6 +116,7 @@ type Notification struct {
 	Tag          string `json:"tag,omitempty"`
 	Color        string `json:"color,omitempty"`
 	ClickAction  string `json:"click_action,omitempty"`
+	AndroidChID  string `json:"android_channel_id"`
```
Comment (luismfonseca) [inline gcm.go]: If the field is not mandatory, we should have `,omitempty` in the json tag.

Comment (hugoghx) [reply gcm.go]: Oh, right!
One could say I .... _omitted that_

**Comment by hugoghx** [comment]
Tested on staging and working. Ready to be merged

**Review by luismfonseca — APPROVED** [review]
LGTM

**Comment by jameshfisher** [comment]
gg

**Pull request merged**